robottelo/logging.py

```diff
@@ -0,0 +1,54 @@
+import os
+from logging import Formatter
+
+import yaml
+from attrdict import AttrDict
+from logzero import setup_logger
+
+from robottelo.config import get_project_root
+
+ROBOTTELO = 'robottelo'
```

Comment:
:) why?

Reply:
You'll see! I was considering modifying the default logzero logger instance, so that the default for the framework is robottelo.log without it having to be specified by name.

Reply:
ahhh so maybe something like this could be done?
```python
from robottelo import logging
logging.ROBOTTELO = f"robottelo_{xdist_id}"
logging.setup...
```